Makefile

```
@@ -79,6 +79,11 @@ else
 		"$(SOURCEDIR)" "$(BUILDDIR)"
 endif
 
+.PHONY: dev-pdf
+dev-pdf: pdf
+	evince build/pdf/wire_federation.pdf 2>&1 > /dev/null &
```

```suggestion
	xdg-open build/pdf/wire_federation.pdf 2>&1 > /dev/null &
```

This is only slightly better than evince as it will not work on mac. Best thing would be to detect a mac and use `open` and on linux use `xdg-open`. (Don't care for Windoze :rofl:)